lib/widgets/login.dart

```diff
@@ -5,6 +5,7 @@ import '../api/exception.dart';
 import '../api/route/account.dart';
 import '../api/route/realm.dart';
 import '../api/route/users.dart';
+import '../model/database.dart';
 import '../model/store.dart';
```

Comment:
It feels a bit unsatisfying to have to say that this file imports our database library, as that should be a layer it doesn't interact with directly.

And there is a handy Dart feature we can use to avoid that, which in fact we already use for a couple of related types. We can say:
```diff
--- lib/model/store.dart
+++ lib/model/store.dart
@@ -29,2 +27,2 @@ import 'stream.dart';
 export 'package:drift/drift.dart' show Value;
-export 'database.dart' show Account, AccountsCompanion;
+export 'database.dart' show Account, AccountsCompanion, AccountAlreadyExistsException;
```

Then this `model/database.dart` import becomes happily redundant.